mods/persistence/game/machinery/machinery.dm

```diff
@@ -11,4 +11,73 @@
 
 /obj/machinery/before_save()
 	. = ..()
-	initial_access = list() // Remove initial_access so that access isn't wiped on load.
\ No newline at end of file
+	initial_access = list() // Remove initial_access so that access isn't wiped on load.
+
+/obj/machinery/power/tracker/after_deserialize()
+	. = ..()
+	connect_to_network() //There's a powernet mismatch sometimes after loading the tracker
+
+
+SAVED_VAR(/obj/machinery, stat)
+SAVED_VAR(/obj/machinery, emagged)
+SAVED_VAR(/obj/machinery, malf_upgraded)
+SAVED_VAR(/obj/machinery, use_power)
+SAVED_VAR(/obj/machinery, uid)
+SAVED_VAR(/obj/machinery, panel_open)
+SAVED_VAR(/obj/machinery, uncreated_component_parts)
+//SAVED_VAR(/obj/machinery, component_parts)
```

Comment:
Maybe add a comment here about the explicit exclusion.

Reply:
It probably just shouldn't be in there tbh. Machinery fixing stuff is on another branch currently. Not sure why I didn't remove that.